app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -171,7 +171,6 @@ def save_key_pairs_inventory(ems, hashes, target = nil)
   def save_cloud_volumes_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    return unless ems.cloud_volumes
```

Comment:
@hsong-rh My comment has nothing to do about tests. This code is common for all cloud providers. All except OpenStack requires this code. My question is does it matter if this code is executed for openstack as well (since you removed the conditional). Will this code do anything for OpenStack?